### PR TITLE
Prepare for 7.0.0 release

### PR DESCRIPTION
**Pull request by h-haaks**
#### Pull Request (PR) description
Preparing for the 7.0.0 release

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -66,3 +66,4 @@
     }
   ]
 }
+
```
Comment (bastelfreak) [inline metadata.json:69]: please remove the empty newline

**Comment by bastelfreak** [comment]
There already is a https://github.com/voxpupuli/puppet-logstash/blob/master/CHANGELOG file with an old changelog. Usually we try to preserve that:
* `git mv CHANGELOG HISTROY.md`
add  `## [6.1.5](https://github.com/voxpupuli/puppet-logstash/tree/6.1.5) (2018-11-13)` to the HISTROY.md
* regenerate the changelog
that should create you a CHANGELOG.md with a new entry for 7.0.0 + the old entries from HISTORY.md

**Comment by h-haaks** [comment]
@bastelfreak do I need to get CHANGELOG -> HISTROY.md merged to master before regenerating the changelog?
I tried doing it all locally but it had no effect ...

**Comment by bastelfreak** [comment]
I created https://github.com/voxpupuli/puppet-logstash/pull/422

**Pull request closed without merging**